en/docs/develop/restapis/applications.yaml

```diff
@@ -0,0 +1,2962 @@
+              schema:
+                $ref: '#/components/schemas/Error'
+        '403':
+          description: Forbidden
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+        '404':
+          description: Not Found
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+        '500':
+          description: Server Error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+        '501':
+          description: Not Implemented
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+    post:
+      tags:
+        - Applications
+      summary: |
+        Add application
+      operationId: createApplication
+      description: >
+        This API provides the capability to store the application information
+        provided by users.
+      parameters:
+        - in: query
+          name: template
+          required: false
+          description: |
+            Pre-defined template to use when creating the application.
+          schema:
+            type: string
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/ApplicationModel'
+        description: This represents the application to be created.
+        required: true
+      responses:
+        '201':
+          description: Successful response.
+          headers:
+            Location:
+              description: Location of the newly created application.
+              schema:
+                type: string
+        '400':
+          description: Bad Request
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+        '401':
+          description: Unauthorized
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+        '403':
+          description: Forbidden
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+        '409':
+          description: Conflict
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+        '500':
+          description: Server Error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+        '501':
+          description: Not Implemented
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+  /applications/import:
+    post:
+      tags:
+        - Applications
+      summary: |
+        Create an application from an exported XML file.
```

Comment:
Lets follow a similar format for all the "summary" fields. Since the first two follow this kind of format ("List applications, Add applications", shall we change this to "Create application from an exported XML file" ?

Please follow the same format in all the other summary fields without the full stop at the end.